huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go

```diff
@@ -645,30 +657,21 @@ func resourceCCENodePoolRead(_ context.Context, d *schema.ResourceData, meta int
 	}
 	mErr = multierror.Append(mErr, d.Set("data_volumes", volumes))
```

Comment:
you can use flattenResourceNodeDataVolume

Reply:
done.